carte_cli/utils/templates/dataset_flattened.md

```diff
@@ -0,0 +1,47 @@
+# {{ dataset.name }} {: .page-title }
```

Comment:
The `{: ... }` tags are Markdown attribute lists, they control how what HTML attributes the compiled markdown will have. More on this: https://python-markdown.github.io/extensions/attr_list/
We can't use straight HTML as that breaks search results in weird ways (mostly likely during term tokenisation).